### ISSUE TITLE
`prevent-abbreviations`: support CSS

### ISSUE THREAD

**Issue by fregante** [sindresorhus/eslint-plugin-unicorn#3265]
### Description

As titled. Honestly I don't know if it's useful or needed, the idea just stemmed from 

- https://github.com/sindresorhus/eslint-plugin-unicorn/issues/3257
- #2967 

I think the classes can benefit from `prevent-abbreviations` if it's a CSS module otherwise the rule might just be too difficult/noisy to follow. Can the rule support CSS Modules only?

### Examples

```css
// ❌
.dev {
  content: 'dev'
}

// ✅
.development {
  content: 'development' /* probably not!? */
}

// ✅
:global(.dev) {
  content: 'dev'
}
```

**Comment by sindresorhus** [comment]
I don't think this is a good idea.

It would really be targeting CSS Modules, which is a non-standard build-tool convention, not actual CSS. `:global()`/`:local()` arent real CSS, and `@eslint/css` doesnt model them. Detection would also rely on a `.module.css` filename convention that breaks for `.module.scss` or for setups that enable Modules on plain `.css`.

More importantly, this rule's value is its autofix: it owns a variable's scope and renames every reference. A CSS class has no scope the linter can see, its references live in other files (`styles.dev`), so this could only ever be a report-only nag you'd fix by hand in two places.
